src/isal/__init__.py

```diff
@@ -18,16 +18,15 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-import pkg_resources
-
 from ._isal import ISAL_MAJOR_VERSION, ISAL_MINOR_VERSION, \
     ISAL_PATCH_VERSION, ISAL_VERSION
 
 __all__ = [
     "ISAL_MAJOR_VERSION",
     "ISAL_MINOR_VERSION",
     "ISAL_PATCH_VERSION",
-    "ISAL_VERSION"
+    "ISAL_VERSION",
+    "__version__"
 ]
 
-__version__ = pkg_resources.get_distribution(__package__).version
+__version__ = "0.3.0-dev"
```

Comment:
I follow semantic versioning (www.semver.org). And I am not going to divert from that.
It is not a problem, because the stable releases (0.1.0, 0.2.0 so far) are correct according to both semver and setuptools.

Reply:
> Why did you revert to hard-coding the version number here? Now you have to remember to update both setup.py and this file before making a release.

I already had to do that in the changelog and setup.py as well. So instead of doing it in two places I now need to do it in 3. There is a reminder in the release procedure. 

The advantage of this is that `setuptools` is no longer a runtime dependency. I like that better as python-isal should be truly standalone.
Also, as I work longer and longer maintaining several packages I think the whole `__version__` thing is a bit redundant. You can see it with `pip list` anyway. The only reason I include it in this particular project is that there is a difference between the version of the package and the version of ISA-L that it is compiled to. It is not that big of a deal. I probably already spent more time writing this comment than I will ever will on updating this version number ;-). So no need to put any code there. Overengineering is a thing too.

Reply:
Fine with me of course. Note that `importlib.resources` became part of the stdlib in Python 3.9, which you can use as a replacement for `pkg_resources` (`pkg_resources` is also very slow).